.github/workflows/ci.yml

```
@@ -0,0 +1,46 @@
+name: Tests
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - develop
+
+jobs:
+
+  pre-commit:
+    name: Pre-commit checks
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        pip install pre-commit==2.2.0
```

```suggestion
      run: pip install pre-commit==2.2.0
```

---

Maybe we should update this version to the latest version of `2.8.2`?
```suggestion
      run: pip install pre-commit==2.8.2
```